spec/acceptance/api/fields_definition_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+resource 'FieldsDefinition' do
+  let(:resource) { FactoryBot.create(:fields_definition,
+                                     account: provider,
+                                     target: 'Account',
+                                     name: 'stuff',
+                                     choices: %w[Orange Apple Banana]) }
```

Comment:
Unimportant, but I think this way is more consistent with the ~rest~ majority of the codebase:
```suggestion
  let(:resource) { FactoryBot.create(:fields_definition, account: provider,
                                                         target: 'Account',
                                                         name: 'stuff',
                                                         choices: %w[Orange Apple Banana]) }
```